public/modules/custom/helfi_rekry_content/helfi_rekry_content.module

```diff
@@ -215,6 +215,15 @@ function helfi_rekry_content_preprocess_field(&$variables) {
 
   // Expose original language to field templates.
   $variables['original_language'] = $originalLanguage;
+  $inlineFields = [
+    'title',
+    'field_salary',
+  ];
+
+  // Use span instead of div for inline fields.
+  if ($variables['field_name'] && in_array($variables['field_name'], $inlineFields)) {
```

Comment:
should the first if condition be `isset($variables['field_name'])`?

Reply:
Yeah, this should be fixed now